node_name1:
  ros__parameters:
    # デバッグ用
    test_show: true
    sleep_time: 0.001

    # 経由点
<<<<<<< HEAD
    way_points_x: [-0.167,16.5,17.4,-15.9,-17,-0.167] ######
    way_points_y: [0.769,-0.323,13.4,15.4,1.54,0.769] ######

    # 車両マージン（ルンバの半径）
    margin_: 0.3
=======
    way_points_x: [] ######
    way_points_y: [] ######

    # 車両マージン（ルンバの半径）
    margin: ######
>>>>>>> 34d755d91456c3df8feffe402ee154d9cd5154c6
